installer/dmg/scripts/postinstall

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+# Variables the installer application automatically passes along
+# $0 : Returns the path to the script
+# $1 : Returns the path to the package
+# $2 : Returns the target location (for example: /Applications)
+# $3 : Returns the target volume (for example: /Volumes/Macintosh HD)
+
+#############
+### Setup ###
+#############
+TARGET_VOLUME=$3
+LOG_FILE="/tmp/postinstall_MongoDB_Atlas_SQL_ODBC.log"
+
+# Log an error
+err(){
+    echo "Err: $@" | tee -a $LOG_FILE
+}
+
+# Log an general info
+info(){
+    if [[ ! -e $LOG_FILE ]]; then
+        touch $LOG_FILE
+    fi
+     echo "Info: $@" | tee -a $LOG_FILE
+}
+
+# Setup the log file
+if [[ ! -e $LOG_FILE ]]; then
+    # Create a new log file
+    touch $LOG_FILE
+else
+    # Clean up the existing file
+    > $LOG_FILE
+fi
+
+ODBC_PATH="/Library/ODBC"
+if [[ ! -z $TARGET_VOLUME ]]; then
+    ODBC_PATH="$TARGET_VOLUME/Library/ODBC"
+fi
+INI="$ODBC_PATH/odbcinst.ini"
+info "Drivers configuration will be added to "$INI""
+INSTALL_ROOT="/Library/MongoDB/MongoDB Atlas SQL ODBC/"
+# Get the latest installed version
+LATEST_VERSION=$(ls "$INSTALL_ROOT/"| grep -E "[0-9]\.[0-9]" | sort | cut -d '/' -f2 | tail -n 1)
+INSTALL_PATH=$INSTALL_ROOT$LATEST_VERSION
+DRIVERS_SECTION="ODBC Drivers"
+MDB="MongoDB Atlas SQL ODBC $LATEST_VERSION"
+MDB_DRIVERS="$INSTALL_PATH/libatsql.dylib"
+CREATED_NEW_INI_FILE=0
+MDB_DRIVER_INSTALLED=0
+MDB_DRIVER_CONFIGURED=0
+
+# Flag the driver as installed if not already done
+flagDriversAsInstalled(){
+        if [ $MDB_DRIVER_INSTALLED == 0 ]; then
+            echo "$MDB = Installed" >> "$INI"
+            MDB_DRIVER_INSTALLED=1
+        fi
+}
+
+#############################################################
+### Configure the newly installed drivers in odbcinst.ini ###
+#############################################################
+# Create the ODBC directory if it doesn't exist already
+if [[ ! -e $ODBC_PATH ]]; then
+    info "Creating $ODBC_PATH directory"
+    mkdir -p $ODBC_PATH
+elif [[ ! -d $ODBC_PATH ]]; then
+     err "$ODBC_PATH already exists but is not a directory"
+     exit 1
+fi
+
+#Update the odbcinst.ini with the latest driver path and mark it as installed
+if [[ ! -e "$INI" ]]; then
+    info ""$INI" does not exists, creating"
+    touch "$INI"
+    echo "[ODBC Drivers]" >> "$INI"
+    echo "$MDB = Installed" >> "$INI"
+    echo "" >> "$INI"
+    echo "[$MDB]" >> "$INI"
+    echo "Driver = $MDB_DRIVERS" >> "$INI"
+    echo "DriverUnicodeType = utf16" >> "$INI"
+    echo "" >> "$INI"
+    $CREATED_NEW_INI_FILE=1
```

Comment:
```suggestion
    CREATED_NEW_INI_FILE=1
```
No need for the `$` since we're setting the value.

Reply:
That's... weird. I didn't write this line